humans/chain.json

```diff
@@ -1,29 +1,25 @@
 {
   "$schema": "../chain.schema.json",
-  "chain_name": "humans",
+  "chain_name": "Humans.ai",
```

Comment:
Please don not change the chain_name. It must match the folder name and it may never change. It's purely an internal key, so interfaces should not be displaying this, but instead the pretty_name

Reply:
Hello @JeremyParish69, my bad I overlooked this aspect, added a new commit to correct this, thank you!